src/main/java/com/blamejared/crafttweaker/impl/brackets/BracketValidators.java

```diff
@@ -170,6 +171,12 @@ public static boolean validateProfessionBracket(String tokens) {
     public static boolean validateResourceBracket(String tokens) {
         return ResourceLocation.tryCreate(tokens) != null;
     }
+    
+    @ZenCodeType.Method
+    @BracketValidator("tooltype")
+    public static boolean validateToolTypeBracket(String tokens) {
+        return tokens.chars().allMatch(c -> ('a' <= c && c <= 'z') || c == '_');
```

Comment:
Is this a defined thing with tooltypes? Couldn't a mod make a tooltype like `pick-axe`?

Reply:
[Only a-z and _ are allowed, meaning names must be lower case. And use _ to separate words.](https://github.com/MinecraftForge/MinecraftForge/blob/c0e74bc8a8ae68ec07c30e86fc11284a18135bec/src/main/java/net/minecraftforge/common/ToolType.java#L28)